cmd/notary/keys.go

```diff
@@ -10,37 +11,81 @@ import (
 	"time"
 
 	"github.com/docker/notary/keystoremanager"
+	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/trustmanager"
 
 	"github.com/spf13/cobra"
 )
 
 func init() {
-	cmdKeys.AddCommand(cmdKeysRemoveRootKey)
-	cmdKeys.AddCommand(cmdKeysGenerateRootKey)
+	cmdKey.AddCommand(cmdKeyList)
+	cmdKey.AddCommand(cmdKeyRemoveRootKey)
+	cmdKey.AddCommand(cmdKeyGenerateRootKey)
+
+	cmdKeyExport.Flags().StringVarP(&keysExportGUN, "gun", "g", "", "Globally unique name to export keys for. A new password will be set for all the keys. Output format is a zip archive.")
```

Comment:
I think it's better to specify - as the output file to make it go to a terminal. Defaulting to outputting binary stuff to stdout is pretty unfriendly. At the very least, we should check whether stdout is a terminal.

Is the following closer to what you're suggesting?

    notary key export -o file.zip docker.com/docker   # export a specific GUN
    notary key export -o file.zip --all   # export all
    notary key export -o file.zip    # return an error becuase you need either a GUN or --all
    notary key export -o file.zip --all docker.com/docker   # return an error because the GUN and --all are mutually exclusive

I'd still rather not have the GUN as a positional option if there are variants where it shouldn't be specified.